app/models/course.rb

```diff
@@ -142,6 +141,13 @@ def students
     course_user_data.where(course_assistant: false, instructor: false, dropped: [false, nil])
   end
 
+  def assessment_categories
+    assessments.pluck(:category_name).uniq.sort
```

Comment:
uniq is an alias for distinct, which seems to be the newer, cooler way to specify this, so maybe go with that?  No big deal.